modules/compliance-tests/src/smithy4s/compliancetests/internals/Assertions.scala

```diff
@@ -29,7 +30,7 @@ private[internals] object assert {
   def fail(msg: String): ComplianceResult = Left(msg)
 
   private def isJson(bodyMediaType: Option[String]) =
-    bodyMediaType.exists(_.equalsIgnoreCase("application/json"))
+    bodyMediaType.forall(_.equalsIgnoreCase("application/json"))
```

Comment:
to be sure: this means that if no `bodyMediaType` is defined, we consider the payload to be json

Reply:
correct